timm/utils/cuda.py

```diff
@@ -17,7 +17,7 @@
 class ApexScaler:
```

Comment:
@rwightman do you mind if we remove the Apex variant of AMP and just use native?